Update service_remapping_rules.md (#35972)
DataDog/documentation: content/en/tracing/services/service_remapping_rules.md

   Alternatively, navigate to **APM** > [**Software Catalog**][14] and click on a service to open the service side panel. From there, click **Service Page** > **Service Remapping**.
   {{< img src="tracing/services/renaming_rules/service-side-panel.png" alt="The side panel for a service, showing the Service Page dropdown menu with a Service Remapping option" style="width:100%;" >}}
1. Choose a remapping action to perform for your new remapping rule.
   - Select **Remap services** to split a single entity, rename an entity, merge multiple entities together, or rename several entities.
   - Select **Correlate telemetry** to identify a service based on an infrastructure tag.
1. Use the search bar to select the entities you want to remap.

>    - You can select one or more entities, but all must be of the same type (service, inferred service, datastore, or queue). Select services based on their `service` or `peer.service` tag, not by their Display Name metadata.

Reviewer: ```suggestion
   - You can select one or more entities, but all must be of the same type (service, inferred service, data store, or queue). Select services based on their `service` or `peer.service` tag, not by their Display Name metadata.
```
Can we provide a reason why they shouldn't use the Display Name metadata? I find that if we can provide some background, it's easier for users to understand and adhere to that guideline.

Author: In the UI we spell datastore as one word

Author: If they use the Display Name metadate it wont work and they wont find the service they are looking for. the search is by tags

   - As you select entities, a span query is built in the background. To edit the query, select **Build Advanced Query**.

>        - Select **Service Rule** to remap services. Select **Inferred Service Rule** to remap inferred services, datastores, or queues.

Reviewer: ```suggestion
       - Select **Service Rule** to remap services, or select **Inferred Service Rule** to remap inferred services, data stores, or queues.
```

       - Select **Add Condition** to add an `AND` condition to your query.

>        - Adding multiple values in the **Value** section creates an `OR` condition.

Reviewer: ```suggestion
       - Add multiple values in the **Value** section to create an `OR` condition.
```

>    - If you're correlating a service with infrastructure tags, you can only select _one_ service. Choose infra tag(s) to correlate telemetry on. All telemetry with the same infra tag(s) as the service chosen will be remapped to a single unified service name.

Reviewer: ```suggestion
   - If you're correlating a service with infrastructure tags, you can only select _one_ service to correllate. Choose infra tags to correlate telemetry on. Datadog remaps all telemetry with the same infra tags as the service chosen to a single unified service name.
```

### Step 2: Specify new entity name

> In the text box, enter a unique name for the selected entity (or entities). Alternatively, use tag values with the `{{tagName}}` syntax to remap based on an entity's tags.

Reviewer: ```suggestion
In the text box, enter a unique name for the selected entity (or entities). Alternatively, use tag values with the `{{tagName}}` syntax to remap based on an entity's tags. As you type, a preview of the new service name(s) appears.
```

   1. If tag values follow a pattern, apply a regular expression to extract only the portion you want in the name.

>    1. View a preview of the new service name(s). Note: The preview is not an exhaustive list. If you are remapping a service based on a tag with several values, only the values with the most spans will appear in the preview. 

Reviewer: ```suggestion
**Note**: The preview is not an exhaustive list. If you are remapping a service based on a tag with several values, only the values with the most spans appear in the preview. 
```

### Step 3: Name your rule and review

1. Optionally, enter a descriptive name for the remapping rule so you can identify it later.

> 1. Review and save your remapping rule. Note: It may take a minute for the remapping rule to take effect.

Reviewer: ```suggestion
1. Review and save your remapping rule. After you save your rule, it may take a minute for it to take effect.
```

## Remapping rules behavior

> Remapping rules work by overriding the `service` tag for remapping services, or the `peer.service` tag for remapping inferred services, datastores, and queues. Services are remapped at intake, and any preexisting configuration specifying a service name does not change when a remapping rule is created. Service remapping rules take precedence over all other service name configurations.

Reviewer: ```suggestion
Remapping rules work by overriding the `service` tag for remapping services, or the `peer.service` tag for remapping inferred services, data stores, and queues. Services are remapped at intake, and any preexisting configuration specifying a service name does not change when a remapping rule is created. Service remapping rules take precedence over all other service name configurations.
```

> Remapping rules are applied across APM, Logs, Metrics, USM, DSM, DJM, DBM, Profiling, NPM, Live Processes, Live Containers, Kubernetes, and Events.

Reviewer: ```suggestion
Remapping rules are applied across APM, Logs, Metrics, USM, DSM, DJM, DBM, Profiling, CNM, Live Processes, Live Containers, Kubernetes, and Events.
```

Author: this should be NPM (network perf monitoring)
